src/components/howTos.stories.mdx

```diff
@@ -0,0 +1,36 @@
+<Meta title="How-tos" />
+
+# How-to guides
+
+There are some particular use cases in this page. You can refer to the usage and some examples below.
+
+## Enabling content share in Electron applications
+
+If you are building a Electron app, you would need to create your own screen picker, and pass the `sourceId` to `useContentShareControls` hook to start content share.
```

Comment:
How about the following to briefly explain why the custom picker is needed?
> In an Electron application, you should create a custom screen picker to retrieve the screen's source ID and pass it to the `useContentShareControls` hook to start sharing your screen.

Reply:
Sure, will update it